Fixed column-moving issue

belskiyvv opened the pull request:
Fixed column-moving issue for tables with horizontal scroll and for IE11.

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/angular-ui/ng-grid/3652)
<!-- Reviewable:end -->

swalters commented:
Hi @belskiyvv could you explain what wasn't working in IE11 and how this fixes it?

belskiyvv commented:
Hi @swalters 
with ".movingColumn { position: absolute;}"  it work fine in all browsers for table without scroll
but if table with scroll, this feature working wrong.
It may be fixed, if set "position: fixed;" . But then it working wrong in IE 11. 

Sorry for my english, please see the this video (column-moving bug in chrome): https://drive.google.com/file/d/0B9IzvGgonTIaZWhtMFJtZVVmOTg/edit
and this gif (IE11 with "position: fixed;"):
http://gyazo.com/2ca6c19e3e674791fdb0f459dde11a3d

With "position: absolute;" bug occurs because .getBoundingClientRect() returns incorrect values
I used .offsetLeft and .offsetWidth properties..

alam123 commented:
Any chance of this PR getting merged?

belskiyvv commented:
@thomashilzendegen, thank you for review. JQuery was removed.

mermonkey commented:
Is this abandoned or still getting integrated? Will it help with #3448? cheers!

tomaszczechowski commented:
Can we merge please? It's quite annoying issue.

Thanks!

AgDude commented:
Closing, as this hasn't seen activity in a long time, tests are failing, and does not meet our single commit per PR requirement.

AgDude commented:
see also #4006
